gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```

Comment:
is this necessary? we don't wanna be using new features which are incompatible with lower versions.

Reply:
I did this initially because I couldn't get the shadow plugin working and didn't know if it was because of issues between Gradle 5.2.1 and 5.6.4, and I didn't bother spending the time to try it again with the older Gradle version. Since this seems important to you, I'll downgrade Gradle and run it through the CI again.